pykickstart/commands/autopart.py

```diff
@@ -76,9 +141,24 @@ def __str__(self):
         return retval
 
     def _getParser(self):
-        op = KSOptionParser()
-        op.add_argument("--encrypted", action="store_true", default=False)
-        op.add_argument("--passphrase")
+        op = KSOptionParser(prog="autopart",  description="""
+Automatically create partitions -- a root (/) partition, a swap
+partition, and an appropriate boot partition for the architecture. On
+large enough drives, this will also create a /home partition.
+
+The ``autopart`` command can't be used with the logvol, part/partition, raid,
+reqpart, or volgroup in the same kickstart file.""")
```

Comment:
Indenting this block would make it look nicer, and I think it'll strip the same # of spaces off when making the string.